.github/workflows/maven-build-skip.yml

```diff
@@ -0,0 +1,29 @@
+#  Copyright 2021 Collate
```

Comment:
We don't need another Workflow just to specify "Step is not required". Github Actions will not run on those PR's which does not have changes in the path's mentioned in Maven CI.

Reply:
As far as I understand it, we are running it even for PRs related to documentation because it is a required action for all of them. It will be skipped when actions are run in the main branch. This is the suggested way to do it by the GitHub [documentation](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks).